src/Platform/Microsoft.Testing.Platform/Resources/PlatformResources.resx

```diff
@@ -713,4 +713,7 @@ Takes one argument as string in the format &lt;value&gt;[h|m|s] where 'value' is
     <value>Test session is aborting due to reaching failures ('{0}') specified by the '--maximum-failed-tests' option.</value>
     <comment>{0} is the number of max failed tests.</comment>
   </data>
-</root>
+  <data name="AbortForMaxFailedTestsCapabilityNotAvailable" xml:space="preserve">
+    <value>The current test framework does not support '--maximum-failed-tests'.</value>
```

Comment:
I am wondering if we should add even more details like they need to open a bug on the test framework repo.

Or change message to be `The current test framework does not implement correctly the maximum failed test capability`